addons/point_of_sale/models/pos_order.py

```diff
@@ -348,10 +352,10 @@ def _default_pricelist(self):
     company_id = fields.Many2one('res.company', string='Company', required=True, readonly=True, default=lambda self: self.env.user.company_id)
     date_order = fields.Datetime(string='Order Date', readonly=True, index=True, default=fields.Datetime.now)
     user_id = fields.Many2one('res.users', string='Salesman', help="Person who uses the cash register. It can be a reliever, a student or an interim employee.", default=lambda self: self.env.uid)
-    amount_tax = fields.Float(compute='_compute_amount_all', string='Taxes', digits=0)
-    amount_total = fields.Float(compute='_compute_amount_all', string='Total', digits=0)
-    amount_paid = fields.Float(compute='_compute_amount_all', string='Paid', states={'draft': [('readonly', False)]}, readonly=True, digits=0)
-    amount_return = fields.Float(compute='_compute_amount_all', string='Returned', digits=0)
```

Comment:
Hmm, I agree with @jorenvo that the backend is somehow irrelevant or even dangerous (black box) to manipulate. It is more important to do reprint (not just the last one but on demand) and refund of any ```pos.order``` in the frontend. If you want to do anything in the backend you should go for ```sale.order``` or ```account.invoice```